.github/ISSUE_TEMPLATE.md

```diff
@@ -5,8 +5,8 @@
 
 ##### Reproduction Steps
 1. [Step 1]
-1. [Step 2]
-1. [Step 3]
+2. [Step 2]
```

Comment:
It renders correctly and means that if you want to insert a new item in the middle of existing items you don't have to rewrite every single number in your list.

```
Ordered lists use numbers followed by periods:

1.  Bird
2.  McHale
3.  Parish

It’s important to note that the actual numbers you use to mark the list have no effect on the HTML output Markdown produces. The HTML Markdown produces from the above list is:

<ol>
<li>Bird</li>
<li>McHale</li>
<li>Parish</li>
</ol>

If you instead wrote the list in Markdown like this:

1.  Bird
1.  McHale
1.  Parish

or even:

3. Bird
1. McHale
8. Parish

you’d get the exact same HTML output. The point is, if you want to, you can use ordinal numbers in your ordered Markdown lists, so that the numbers in your source match the numbers in your published HTML. But if you want to be lazy, you don’t have to.

If you do use lazy list numbering, however, you should still start the list with the number 1. At some point in the future, Markdown may support starting ordered lists at an arbitrary number.
```

From http://daringfireball.net/projects/markdown/syntax#list